### ISSUE TITLE
Style: improve the look of the read aloud / "playback" button

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/docassemble-AssemblyLine#29]
There are some examples of more unobtrusive widgets that do this around the web. We should be able to style this better.

MLRI's is one good example: 
![image](https://user-images.githubusercontent.com/7645641/108247635-e7654d00-7120-11eb-9999-c976ef3d3e46.png)

What I like about this is that it has a label and icon that more clearly conveys what it does and that the button does not take the full width of the CSS box.

**Comment by nonprofittechy** [comment]
@purplesky2016 has some experience trying to customize the MassLegalHelp version of this input.

@CaroRob might have the model website she liked better

**Comment by nonprofittechy** [comment]
This can be started now--it will be useful. Or it can wait for an interested designer volunteer.

**Comment by purplesky2016** [comment]
I made an attempt, see https://github.com/SuffolkLITLab/docassemble-AssemblyLine/pull/43.

**Comment by nonprofittechy** [comment]
This definitely looks better! @purplesky2016 do you think we can add in a text label to the input with CSS as well?

**Comment by purplesky2016** [comment]
Could you elaborate it a bit?

**Comment by BryceStevenWilley** [comment]
Do you mean something like adding the words "Read the screen" in front of the play button @nonprofittechy? I do think that would be a little better.

**Comment by purplesky2016** [comment]
If we are talking about labels like the following for those radio buttons
![image](https://user-images.githubusercontent.com/20231649/110151103-0d8c1d80-7dae-11eb-985e-f3fc629409cb.png)
we need to be able to control the "input" buttons. It appears that the "audio" button is not directly under our control out of box.

**Comment by purplesky2016** [comment]
This is what it looks like in MS Edge:
![image](https://user-images.githubusercontent.com/20231649/110151979-1fba8b80-7daf-11eb-8581-530f725d9f37.png)

**Comment by plocket** [comment]
I'm going to give a quick [hybrid js/css solution](https://stackoverflow.com/a/39899797/14144258) a shot and see what you folks think. It's looking like it's the standard way people out there handle better looking audio UI.

**Comment by plocket** [comment]
We're choosing to leave out the option of rewinding/scanning back manually, correct? I'm using this as the basic inspiration: https://www.readspeaker.com/

**Comment by plocket** [comment]
Needs testing in different browsers: https://github.com/SuffolkLITLab/docassemble-AssemblyLine/tree/29_custom_audio_2

**Comment by nonprofittechy** [comment]
These are all good steps forward and are nicer than the built-in appearance. I'm not sure I'm a fan of any of the improvements yet, but I do think the use of JS will be necessary to fully improve it.

![image](https://user-images.githubusercontent.com/7645641/110491354-33bc0100-80bf-11eb-8078-b3d608e7a4a3.png)

Overall, I like this. A few thoughts as I tested it out:

* My eye goes right to the audio icon and the "listen" text, but they are not interactive. I tried to click the "listen" text because it looked like it should be a button, but nothing happened.
* The play/stop/volume icons seem oddly large relative to the other visual elements.
* The stop icon and volume up/down buttons feel unnecessary.
* The element is very large vertically compared to other elements on the page. My first instinct is that it should be about half the height of the "Next" button. It also has a lot of white space underneath it, pushing the headline "What is your name?" to about halfway down the page on the dev site.

I think this could be made shorter vertically and simplified so the audio icon, the listen text, and the play triangle all start the audio, and the play icon changes to a pause icon (as it already does) to stop playback. There could be a good argument to have a volume slider still present (using the icon on the left might make sense), but the buttons are non-standard. I think we probably don't need a volume adjustment specific to this input.

**Comment by plocket** [comment]
Sounds good. Would love for this to be usable outside of da, as I said, but we can only do what we can do.

Quick question - I don't think I understand what you mean by da vs. bootstrap styles. It has seemed to me that the styles da uses are mainly through bootstrap. Are you referring to something else?

**Comment by plocket** [comment]
The latest commit removed the volume controls, merged the 'listen' with 'play', and reduced the size of the buttons. I double checked and it looks like the whitespace between the element and the `question` heading is actually the same height from what I can tell, not increased, but I remember having the same reaction, so there must be something optical going on that really makes the whitespace stand out now. I halved the amount of that margin.

I've created #52 in case that covers MVP.
